### PR TITLE
Documentation updates.

### PR DESCRIPTION
**Pull request by setrofim**

### CODE REVIEW

File: doc/source/how_tos/users/agenda.rst [hunk 1]
```diff
@@ -504,206 +504,185 @@ turn override global settings.
...
+(e.g. your config file and the config section of the agenda). These entries will
+be combined to form the final configuration for the augmentation used during the
+run. If different values for the same parameter are present in multiple entries,
+the ones "more specific" to a particular run will be used (e.g. values in the
+agenda will override those in the config file).
+
+.. note:: Creating an entry for an augmentation alone does not enable it! You
+          **must** list it under ``augmentations`` in order for it to be enabed
+          for a run. This makes it easier to quickly enabled and diable
+          augmetnations with complex configurations, and also allows defining
```
Comment (marcbonnici) [inline doc/source/how_tos/users/agenda.rst]: augmetnations -> augmentations

File: doc/source/how_tos/users/agenda.rst [hunk 2]
```diff
@@ -504,206 +504,185 @@ turn override global settings.
...
+          "static" configuation in top-level config, without actually enabling
+          the augmentation for all runs.
+
+
+Disabling augmentations
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Sometimes, you may wish to disable an augmentation for a particular run, but you
+want to keep it enabled in general. You *could* modify your config file to
+t temporarily disable it. However, you must then remember to re-enable it
```
Comment (marcbonnici) [inline doc/source/how_tos/users/agenda.rst]: Extra "t"

File: doc/source/how_tos/users/agenda.rst [hunk 3]
```diff
@@ -504,206 +504,185 @@ turn override global settings.
...
+that do not generate their own metrics, ``fps`` instrument is enabled to get
+some meaningful performance metrics for them; the only exception is
+``dhrystone`` which is a benchmark that reports its own metrics and has not GUI,
+so the instrument is disabled for it using ``~fps``.
+
+Each workload will be run in two configurations: once, to collect energy
+measurements, and once to collect thermal data and kernel trace. Trace can give
+insight into why a workload is using more or less energy than expected, but it
+can be relatively intrusive and might impact absolute energy and performance
+metrics, which is why it is collected separately. Classifiers are used to
```
Comment (marcbonnici) [inline doc/source/how_tos/users/agenda.rst]: Would it be worth linking back to the `classifiers` section here?

**Comment by setrofim** [comment]
Comments addressed.

**Pull request merged**